.github/workflows/docs.yml

```diff
@@ -0,0 +1,16 @@
+name: docs
+
+on: [push, pull_request]
```

Comment:
```suggestion
on: [push]
```

Reply:
@collinprather Thanks for catching this. I agree we're running the duplicate tests for no good reason. Since you weren't able to make a suggestion to `test.yml` also (silly for github to only let you make suggestions near code that was changed imo), i'm going to fix all 3 workflow files in a new commit and will mark this as resolved.